### DISCUSSION TITLE
CoE Startkit and App/flow archiving to SharePoint library

### DISCUSSION THREAD

**Discussion by Topkapi77** [microsoft/powerapps-tools#1171] [General]
Hi, In one of the earlier version of CoE, the Governance solution had tools available to archive apps and flows to a SharePoint document library. I can see this in the official youtube video https://youtu.be/6bfaFsFtLow. This feature is not in the latest version. Why was this removed. I think this is a good feature which I would like to have.

**Comment by manuelap-msft** [comment]
This repo is no longer used or monitored by the CoE starter kit team. The new repo where you can raise issues is https://github.com/microsoft/coe-starter-kit/issues/new/choose 

We have updated all our docs to point to the new repo; please update your bookmarks if you still have the old link. Thank you.
